packages/go_router/lib/src/parser.dart

```diff
@@ -24,9 +26,11 @@ class GoRouteInformationParser extends RouteInformationParser<RouteMatchList> {
         redirector = redirect;
 
   /// The route configuration for the app.
+  @internal
```

Comment:
Should this also be added to fields like `redirector` or `debugRequireGoRouteInformationProvder`?

Reply:
Since people will get warning when they import source file directly to use this class. I think adding this annotation is redundant. May as well remove this.

Reply:
Yeah I don't think this annotation is necessary here.